app/tag-management/new-tag/new-tag.component.html

```diff
@@ -0,0 +1,42 @@
+  <div class="top-content">
+    <div class="valign">
+      <md-card>
+        <md-card-title>
+          <h3>{{ 'create new tag' | translate}}</h3>
+        </md-card-title>
+        <md-card-content>
+          <md-grid-list cols="5">
+            <div class="form-wrap">
+              <form novalidate (ngSubmit)="addTag()" #newtag="ngForm">
+                <div *ngIf="errorMessage" class="alert alert-danger">
+                  <span>{{ errorMessage }}</span>
+                </div>
+                <md-input [(ngModel)]="tag.name"  required minlength="1" type="text" placeholder="{{ 'enter tag name' | translate}}"
+                #Name="ngModel" name="Name">
+              </md-input>
+              <md-input [(ngModel)]="tag.shortName"  required minlength="1" type="text" placeholder="{{ 'enter tag short name' | translate}}"
+              #ShortName="ngModel" name="ShortName">
+            </md-input>
+            <md-input [(ngModel)]="tag.description" required minlength="1" type="text" placeholder="{{ 'enter tag description' | translate}}"
+            #Description="ngModel" name="Description">
+          </md-input>
+          <p><label class="text">{{ 'select layer' | translate }}</label>
+            <select class="btn btn-default dropdown-toggle" (change)="selectLayer($event.target.value)">
+              <option *ngFor="let layer of layers" [selected]="layer == layer">{{ layer }}</option>
+            </select></p>
+            <p><input class="right" [(colorPicker)]="tag.style" 
+             [style.background]="tag.style" 
+             placeholder="{{ 'select color' | translate}}" name="Style"/></p>
```

Comment:
The color picker appears next to the field and is not visible anymore in a small window. Please consider that not everyone has a full-HD monitor ;)